index.html

```diff
@@ -169,5 +169,25 @@ <h2 class="more_det">LET’S CONNECT</h2>
              </ul>
          </div>
          </section>
+         <section class="form-bui">
+            <h3 class="my-contact">Contact me</h3>
+            <p class="my_info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/xeqvdqoj">
```

Comment:
- [ ] Kindly add method property with "POST" as the value in order to make the form send data to the action URL.